client/trino-jdbc/src/main/java/io/trino/jdbc/TrinoResultSet.java

```diff
@@ -239,15 +273,9 @@ protected Iterable<List<Object>> computeNext()
                 QueryStatusInfo results = client.currentStatusInfo();
                 progressCallback.accept(QueryStats.create(results.getId(), results.getStats()));
                 warningsManager.addWarnings(results.getWarnings());
-                Iterable<List<Object>> data = client.currentData().getData();
-
-                try {
-                    client.advance();
-                }
-                catch (RuntimeException e) {
-                    throw e;
```

Comment:
That's a good change, but unrelated to the other changes.
Can you move it to separate commit (same or separate PR)?

Reply:
OK, reverted.